import_export/static/import_export/action_formats.js

```diff
@@ -1,22 +1,32 @@
-(function($) {
-  $(document).ready(function() {
-    var $actionsSelect, $formatsElement;
-    if ($('body').hasClass('grp-change-list')) {
-        // using grappelli
-        $actionsSelect = $('#grp-changelist-form select[name="action"]');
-        $formatsElement = $('#grp-changelist-form select[name="file_format"]');
-    } else {
-        // using default admin
-        $actionsSelect = $('#changelist-form select[name="action"]');
-        $formatsElement = $('#changelist-form select[name="file_format"]').parent();
-    }
-    $actionsSelect.change(function() {
-      if ($(this).val() === 'export_admin_action') {
-        $formatsElement.show();
-      } else {
-        $formatsElement.hide();
-      }
+(function ($) {
+    $(document).ready(function () {
+        var $actionsSelect, $formatsElement;
+        $actionsSelect = $('.actions select[name="action"]');
+        $formatsElement = $('.actions select[name="file_format"]');
+        if ($('body').hasClass('grp-change-list')) {
+            // using grappelli
+            $ShowElementToShow = $formatsElement;
+        } else {
+            // using default admin
+            $formatsElementToShow = $formatsElement.parent();
+        }
+        for (let i = 0; i < $actionsSelect.length; i++) {
+            $actionsSelect.eq(i).change(function () {
+                if ($(this).val() === 'export_admin_action') {
+                    $formatsElementToShow.eq(i).show();
+                } else {
+                    $formatsElementToShow.eq(i).hide();
+                }
+            });
+            $actionsSelect.eq(i).change();
```

Comment:
Does this line do anything?  AFAIK it is redundant?  Am I wrong?

Reply:
yes, agree with you
I just keep it, as it was existing before the cahnges as `$actionsSelect.change();`